### PR TITLE
#5/info card

### PR DESCRIPTION
**Pull request by ferdousara-parvin**
Created UI tests, integration tests and unit tests for this story, One integration test is incomplete because of time constraints.

### CODE REVIEW

File: app/src/androidTest/java/com/example/concordia_campus_guide/InfoCardFragmentIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+                                0),
+                        isDisplayed()));
+        textView.check(matches(withText("Engineering, Computer Science and Visual Arts Integrated Complex")));
+    }
+
+    /**
+     * This test validates that the expanded info card contains the correct departments, services and address
+     * @throws UiObjectNotFoundException
+     * @throws InterruptedException
+     */
```
Comment (Hambrsoom) [inline app/src/androidTest/java/com/example/concordia_campus_guide/InfoCardFragmentIntegrationTest.java]: @Test missing

File: app/src/androidTest/java/com/example/concordia_campus_guide/InstrumentalTests/InfoCardFragmentViewModelTest.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import org.junit.Rule;
+import org.junit.Test;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertNotNull;
+import androidx.test.filters.SmallTest;
+import androidx.test.rule.ActivityTestRule;
+import androidx.test.runner.AndroidJUnit4;
+import org.junit.Before;
+import org.junit.runner.RunWith;
+
```
Comment (Hambrsoom) [inline app/src/androidTest/java/com/example/concordia_campus_guide/InstrumentalTests/InfoCardFragmentViewModelTest.java]: What are u testing here ?

File: app/src/main/res/mockito-extensions/org.mockito.plugins.MockMaker [hunk 3]
```diff
@@ -0,0 +1 @@
+mock-maker-inline
```
Comment (viveanban) [inline app/src/main/res/mockito-extensions/org.mockito.plugins.MockMaker]: what is this for?

Comment (viveanban) [reply app/src/main/res/mockito-extensions/org.mockito.plugins.MockMaker]: if it's not needed, we should put it in the gitignore

**Pull request merged**